casper/src/main/scala/coop/rchain/casper/util/rholang/InterpreterUtil.scala

```diff
@@ -37,12 +37,12 @@ object InterpreterUtil {
       dag: BlockDagRepresentation[F],
       runtimeManager: RuntimeManager[F]
   ): F[BlockProcessing[Option[StateHash]]] = {
-    val preStateHash    = ProtoUtil.preStateHash(b)
-    val tsHash          = ProtoUtil.tuplespace(b)
-    val deploys         = ProtoUtil.deploys(b)
-    val internalDeploys = deploys.map(InternalProcessedDeploy.fromProcessedDeploy)
-    val timestamp       = b.header.timestamp
-    val blockNumber     = b.body.state.blockNumber
+    val incomingPreStateHash = ProtoUtil.preStateHash(b)
```

Comment:
Since `replayBlockDeploys` is only called in `validateBlockCheckpoint`, it makes sense to rename the `preStateHash` parameter in `replayBlockDeploys` to `incomingPreStateHash`.

Reply:
That parameter is replaced with the `block` itself in one of the last commits.